### ISSUE TITLE
SVG Image format not supported by GMail

### ISSUE THREAD

**Issue by ShubhankarKG** [CircuitVerse/CircuitVerse#964]
**Describe the bug**
The custom emails that CircuitVerse sends to people when they register, do stuff etc. have a slight bug. When the emails are opened in GMail App/Browser, the CircuitVerse logo fails to render and we see an alternate hyperlink instead.

I have read about this online and the problem is due to the fact that GMail **does not** support SVG extension files. This is not the case with any other browser/App, only GMail.

Since a huge student base relies on Gmail, I think it will be better if we address this issue now. A considerable amount of work is already been done but if the custom emails don't display CircuitVerse, we know how much turned off students can get.

**To Reproduce**
1. Sign Up on the website (with a Google account).
2. You'll receive an email.
3. Open it in the Gmail browser.

**Expected behavior**
![Screenshot (27 Feb 2020 18_54_33)](https://user-images.githubusercontent.com/50791000/75449994-06a91e80-5994-11ea-98bf-1507382d5608.png)

*as displayed from my mobile email app.

**Screenshots**
![Screenshot from 2020-02-27 18-56-44](https://user-images.githubusercontent.com/50791000/75450067-24768380-5994-11ea-976c-eba4ff5a0059.png)

*as seen from the gmail website

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04 LTS
 - Browser: Chrome
 - Version [e.g. 22] : *

**Additional context**
It would be better if we update our database with png file formats. We don't want to mess up the hard work behind this with a petty email issue.

**Are you working on this issue? (Yes/No)**
Yes, I can work. But i would need some help going along.

Please review @tachyons , @ankitkataria

**Comment by MrBartusek** [comment]
Good point but. that should be considered as a separate issue, more like a part of #845

**Comment by Shivansh2407** [comment]
I would like to work on this issue .

**Comment by ShubhankarKG** [comment]
Well i myself could try. I did ths=is to bring it to notice. Hope you don't mind.

**Comment by Shivansh2407** [comment]
No Problem @ShubhankarKG . Open up a PR soon and let me know if you need any help 👍 .

**Comment by ShubhankarKG** [comment]
Thanks, I would want some help @Shivansh2407 , @MrBartusek . Hope y'all will help me. I intend to start from Saturday since I have my exams ongoing.

**Comment by Shivansh2407** [comment]
> Thanks, I would want some help @Shivansh2407 , @MrBartusek . Hope y'all will help me. I intend to start from Saturday since I have my exams ongoing.

Yes Obviously, I am ready to help 👍

**Comment by ShubhankarKG** [comment]
Where is the main image being processed at ? I mean the CircuitVerse logo. I couldn't find it in `app\mailers` or `app\views`. Any ideas @MrBartusek @Shivansh2407

**Comment by aayush-05** [comment]
You can find the new logo in app/assets/images @ShubhankarKG

**Comment by MrBartusek** [comment]
@ShubhankarKG ask on slack

**Comment by aayush-05** [comment]
Can this be closed @nitin10s @tachyons , seems more like a part of #845 ?
